### PR TITLE
pretty README.md

### PR DESCRIPTION
**Pull request by MartinDelille**
Here is a proposal for a nicer README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,45 +1,49 @@
...
+
+![Audacity](images/AudacityLogo.png)
+
 Audacity(R): free, open source, cross-platform audio software for 
-multi-track recording and editing: https://www.audacityteam.org/ . 
+multi-track recording and editing: <https://www.audacityteam.org>
 
 User support is provided on Audacity Forum:
-https://forum.audacityteam.org/ .
+<ttps://forum.audacityteam.org>.
```
Comment (trebmuh) [inline README.md]: missing a "h" letter here

File: README.md [hunk 2]
```diff
@@ -1,45 +1,49 @@
...
 
 Contents of this README:
 
 1.  Licensing
-2.  Changes since version 2.2.2
-3.  Known Issues at Release
-4.  Source Code, Libraries and Additional Copyright Information
+1.  Changes since version 2.2.2
+1.  Known Issues at Release
+1.  Source Code, Libraries and Additional Copyright Information
```
Comment (trebmuh) [inline README.md]: strange that they're all numbered "1" ?

Comment (MartinDelille) [reply README.md]: this is a convenient way markdown provide to auto number the list. We could keep 1. 2. 3. 4 for people that don't have markdown viewer thus.

**Comment by MartinDelille** [comment]
@trebmuh It looks like we are involved in the same projects :-)

File: README.md (hunk 2 above)
Comment (trebmuh) [reply README.md]: @MartinDelille: do you mean that it's only wrongly displayed here in the commit, but will auto-magically display 1., 2.,... when merged?

Comment (MartinDelille) [reply README.md]: No only in the viewer. Try to copy/paste the code here: http://markdownlivepreview.com/

Comment (trebmuh) [reply README.md]: OK

**Comment by JamesCrook** [comment]
Rejected.
Travis build fails with this because Linux make files were not updated.

**Comment by MartinDelille** [comment]
@JamesCrook this PR was just about documentation... I don't understand why I should update the documentation of a plateform I don't use... 😢

**Comment by SteveDaulton** [comment]
If Audacity fails to build on Windows, macOS or Linux, then that is automatically a P1 bug (priority 1).
If you don't fix that, then someone else would have to do so straight away because it blocks all testing on the affected platform.

Personally, I like md files with markup, but I think this change would need to be discussed and agreed by the team. IF it is agreed that we want to change from plain text to markup (and there's pros and cons to both), then additional changes are required beyond what is included in these commits. For example, I think that the file MUST read correctly in a normal text viewer / text editor, because most users that download Audacity will not be viewing the README with a markdown viewer. So, as a specific, I think that numbered lists must retain sequential numbering.

Also, the README.txt file is used in the Windows installer as InfoAfterFile=..\README.txt
According to http://www.jrsoftware.org/ishelp/index.php?topic=setup_infobeforefile it must be " in .txt or .rtf (rich text) format".

So to answer your question briefly, this is not just a cosmetic change in the documentation. It would need to be discussed, agreed, fixed and tested. If you are interested in pursuing this, then the first step is to raise the question on the developer's mailing list (as James has already noted: https://github.com/audacity/audacity/milestone/1 )

**Comment by trebmuh** [comment]
That make sense.

**Comment by MartinDelille** [comment]
Closing the pull request is not cool, especially for a first contribution.

I didn't realized that the README.txt was a build dependency and could have tried to fix that (like I did [here](https://github.com/MartinDelille/audacity/commit/0c876c2485bedaf642d225d963b813f4cd753c86) but since the PR is closed, there is no more travis build)

I correct the numbered list [here](https://github.com/MartinDelille/audacity/commit/8168a1c1722ca4765d15a17158d86136bf49536d)

I did spoke about this change in the mailing list.

**Comment by JamesCrook** [comment]
Martin, now perhaps you understand better why discussion on the mailing list is required?

Your pull request simply had too many problems.  It was an un-necessary un-asked-for format change that creates several problems.  Please read over Steve's comments, and mine.  I agree with everything Steve says there.  If a pull request breaks the build it's a candidate for speedy rejection.

312 is by the way not a first pull request from you.  305 and 309 are still pending.  You have also been given and accepted write access to https://github.com/audacity/portaudio so that, if you want to, you can experiment with making our patched version of it into a submodule.  It would be strange to have wxWidgets as a submodule (pull 305) and not portaudio.

**Comment by MartinDelille** [comment]
Ok sorry for that. I'll be respecting the project policies in the futur.

**Pull request closed without merging**